cli/src/main/kotlin/OrtMain.kt

```diff
@@ -208,7 +208,7 @@ class OrtMain : CliktCommand(name = ORT_NAME, invokeWithoutSubcommand = true) {
 
         """
              ______________________________
-            /        \_______   \__    ___/ the OSS Review Toolkit, version $version.
+            /        \_______   \__    ___/ The OSS Review Toolkit, version $version.
```

Comment:
BTW, originally the letter was deliberately lower-cased, as it was supported to read like "ORT, the OSS Review Toolkit":